sigmastate/src/main/scala/sigmastate/SigSerializer.scala

```diff
@@ -137,7 +137,7 @@ object SigSerializer {
           }
 
           // Verifier doesn't need the polynomial anymore -- hence pass in None
```

Comment:
Comment is outdated, "tests" kind of PR contains undocumented changes in logic.

Reply:
Please be more specific, so I can document what is missing.

Reply:
@aslesarenko code change under this comment is not documented

Reply:
@kushti  The decision to use None was a mistake in a first place, which was not found earlier because the tests where not exhaustive.
Now it is found and fixed, so the code looks like it is expected.
I'm not sure which comment to put in the code. 
I can put this comment in the code, but it will look awkward.

Reply:
but comment still says "// Verifier doesn't need the polynomial anymore -- hence pass in None"